WebRoot/js/ajax/core/AjxImg.js

```diff
@@ -69,12 +69,7 @@ function(parentEl, imageName, useParentEl, _disabled, classes, altText) {
     }
 
     // Get image data based on image name
-    var imageData = window.AjxImgData[AjxImg.getClassForImage(imageName)];
-    if(!imageData) {
-        DBG.println(AjxDebug.IMAGES, "missing image: ", AjxImg.getClassForImage(imageName));
-
-        return;
-    }
+    var imageData = window.AjxImgData[AjxImg.getClassForImage(imageName)] || {};
```

Comment:
can we remove chat icons from repo as well here?
we can ask zextras to modify their code after that

Reply:
Here is the PR for that: https://github.com/Zimbra/zm-web-client/pull/215
Thanks

Reply:
Thanks